scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -152,6 +154,7 @@ package object dialects {
     .withAllowStarWildcardImport(true)
     .withAllowProcedureSyntax(false)
     .withAllowDoWhile(false)
+    .withAllowPlusMinusBeforeUnderscore(true)
```

Comment:
In Scala3 it will work under `-Ykind-projector:underscores` but I think it's safe to enable it by default

Reply:
I wonder if we should have Scala32 dialect for it?

Reply:
Scala3 compiler doesn't accept such syntax without this flag.
It's a kind of `less-restrictive` setting by default.

Reply:
@tgodzik Scala32 will accept underscores for type lambdas, but the `+_`/`-_` will forever be optional under a flag (they make no sense in Scala 3 since lambda parameter variance is inferred, they are parsed so that Scala 2 code can compile, but the marker itself is ignored)